### PR TITLE
enable building of MPI libraries in VTK 8.1.0 easyconfigs

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2585.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0c14f96f52d2509e6b38f3784c8e49fc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2034.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e4aa1e886f15277c0b996edab4166fba for a full test report.

File: easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-2.7.14.eb [hunk 1]
```diff
@@ -44,7 +45,8 @@ dependencies = [
     ('X11', '20180131'),
 ]
 
-configopts = "-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s " % SHLIB_EXT
+configopts = "-DVTK_Group_MPI:BOOL=ON "
```
Comment (damianam) [inline easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-2.7.14.eb]: Why this particular order? I'd suggest to group by either functionality or alphabetically. Now it seems random.

File: easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-2.7.14.eb [hunk 2]
```diff
@@ -44,7 +45,8 @@ dependencies = [
...
+configopts += "-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s " % SHLIB_EXT
 configopts += "-DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.%s " % SHLIB_EXT
 configopts += "-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include -DVTK_WRAP_PYTHON=ON "
```
Comment (damianam) [inline easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-2.7.14.eb]: Divide in 2 lines?

File: easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-3.6.4.eb [hunk 3]
```diff
@@ -44,7 +45,8 @@ dependencies = [
     ('X11', '20180131'),
 ]
 
-configopts = "-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s " % SHLIB_EXT
+configopts = "-DVTK_Group_MPI:BOOL=ON "
```
Comment (damianam) [inline easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-3.6.4.eb]: Same comments as previous file.

File: easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-2.7.14.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-2.7.14.eb]: @damianam Like this it's consistent with the existing VTK 8.1.1 easyconfig, see https://github.com/easybuilders/easybuild-easyconfigs/pull/6404/files#diff-0ee45fd4c5d93b140383c1df76708f6eR48, so I prefer leaving it like this.

I have no particular reason to order the options like this though, so if you insist I'll just tweak the VTK 8.1.1 easyconfig as well to match it.

Comment (damianam) [reply easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-2.7.14.eb]: I think it makes sense to use a sensible order, now that you are changing the easyconfig.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2014.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/434c1fdad8560afb29d7f1567669c419 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/02b7a8b153e5c2cfe684c5af2bc9e512 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2476.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3df599ac7af44cf60072905002ace04c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2041.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/606138d8f28e59867b0ea42c6f76d3d8 for a full test report.

**Pull request merged**